apps/web/src/components/(dashboard)/avatar/stack-avatar.tsx

```diff
@@ -0,0 +1,36 @@
+import { Avatar, AvatarFallback } from '@documenso/ui/primitives/avatar';
+
+export type StackAvatarProps = {
+  first?: boolean;
+  zIndex?: string;
+  fallbackText?: string;
+  type: 'unsigned' | 'waiting' | 'completed';
+};
+
+export const StackAvatar = ({ first, zIndex, fallbackText, type }: StackAvatarProps) => {
+  let classes = '';
+  switch (type) {
+    case 'unsigned':
+      classes = 'bg-dawn-200 text-dawn-900';
+      break;
+    case 'waiting':
+      classes = 'bg-water text-water-700';
+      break;
+    case 'completed':
+      classes = 'bg-documenso-200 text-documenso-800';
+      break;
+    default:
+      break;
+  }
+
+  return (
+    <Avatar
+      className={`
+        ${zIndex && `z-${zIndex}`} 
```

Comment:
This won't work the way you think it might, it'll likely get purged in a production build.